pkg/pgmodel/upgrade_tests/upgrade_test.go

```diff
@@ -107,16 +108,25 @@ func getUpgradedDbInfo(t *testing.T) (upgradedDbInfo dbInfo) {
 
 func getPristineDbInfo(t *testing.T) (pristineDbInfo dbInfo) {
 	withNewDBAtCurrentVersion(t, *testDatabase,
+		/* preUpgrade */
 		func(container testcontainers.Container, _ string, db *pgxpool.Pool, tmpDir string) {
-			defer db.Close()
 			ingestor, err := pgmodel.NewPgxIngestor(db)
 			if err != nil {
 				t.Fatalf("error connecting to DB: %v", err)
 			}
+			defer ingestor.Close()
 
-			doIngest(t, ingestor, preUpgradeData1, preUpgradeData2, postUpgradeData1, postUpgradeData2)
+			doIngest(t, ingestor, preUpgradeData1, preUpgradeData2)
+		},
+		/* postUpgrade */
+		func(container testcontainers.Container, _ string, db *pgxpool.Pool, tmpDir string) {
+			ingestor, err := pgmodel.NewPgxIngestor(db)
+			if err != nil {
+				t.Fatalf("error connecting to DB: %v", err)
+			}
+			defer ingestor.Close()
```

Comment:
why is there an upgrade here? This version is specifically _not_ supposed to have any upgrades, and just go immediately to the final version

Reply:
(if you want to upgrade the extension independant of the connector, that should be a separate test)

Reply:
(if it's just a restart, comments are wrong)

Reply:
Fixed comment